src/utils/middlewares/__tests__/azure_api_auth.test.ts

```diff
@@ -256,3 +262,107 @@ describe("AzureApiAuthMiddleware", () => {
     }
   });
 });
+
+describe("AzureAllowBodyPayloadMiddleware", () => {
+  it("should success if pattern is not matched", async () => {
+    const headers = {
+      ...someHeaders
+    };
+    const aPayload = { foo: { bar: "baz" } };
+    const mockRequest = {
+      header: jest.fn(lookup(headers)),
+      body: aPayload
+    };
+    const aNonMatchingCodec = t.interface({ anyField: t.number });
+    const anyGroupSet = new Set([UserGroup.ApiMessageWrite]);
+
+    const middleware = AzureAllowBodyPayloadMiddleware(
+      aNonMatchingCodec,
+      anyGroupSet
+    );
+
+    const result = await middleware(mockRequest as any);
+
+    expect(isRight(result)).toBe(true);
+    expect(aNonMatchingCodec.decode(aPayload).isLeft()).toBe(true); // test is wrong if it fails
+  });
+
+  it("should success if pattern is not matched", async () => {
```

Comment:
Repeated test name. It can leads to strange behaviors

```suggestion
  it("should success if pattern is not matched - test nr. 2", async () => {
```

Reply:
I committed it, it was a good suggestion.